node-server/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# --------------- BUILDER IMAGE
+
+FROM node:20-alpine as builder
+
+WORKDIR /app
+
+COPY package.json .
+
+# Uncomment if you use a different package manager
+COPY package-lock.json . # npm
```

Comment:
This is not going to work, since we are building images and not just template Dockerfiles. I suggest to change this to something like `koa`, `fastify` or any other frameworks that we might use directly. Don't see many advantages of a node server image